.circleci/config.yml

```diff
@@ -224,7 +223,7 @@ commands:
           name: Store build archive
           command: |
             set -u
-            readvault.sh secret/pepper/${ENVIRONMENT}/v1/conf .data.gcp.serviceKey | gcloud auth activate-service-account --key-file=-
+            readvault.sh secret/pepper/dev/v1/conf .data.gcp.serviceKey | gcloud auth activate-service-account --key-file=-
```

Comment:
As per our chat, the plan here is to switch this to `prod` whenever we build code, since the location of the artifact will always be the production bucket.